Resources/Prototypes/Entities/Items/Clothing/hats.yml

```diff
@@ -2601,6 +2601,19 @@
     type: Clothing
   parent: HatBase
 
+- type: entity
+  id: HatSkub
+  name: Skub Hat
+  description: 'Huh, Skub.'
+  components:
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Sprite
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Icon
+  - sprite: Clothing/Head/skubhead.rsi
+    type: Clothing
+  parent: HatBase
```

Comment:
put the parent above the id

Reply:
> 
> 
> put the parent above the id

Done but idk why you want this. Everything else in the yaml has a parents below the id...